analysis_options.yaml

```diff
@@ -1 +1,58 @@
-include: package:effective_dart/analysis_options.yaml
```

Comment:
Why remove effective dart? :(

Reply:
Initially, I replaced it with `pedantic` because the GitHub Action was failing with it. Then I came across the same issue with `pedantic`.

It seems to be an issue in the `flutter analyze` command: it exits with `code 1` even when there are only warnings - [see the logs](https://github.com/arthurdenner/theme_mode_handler/runs/2045879652?check_suite_focus=true).

Since there isn't much to analyze, it was easier to replace the whole lint file with `pedantic@1.11.0`'s content directly.
If you know any workaround, I appreciate it. I tried `flutter analyze --no-fatal-warnings` too but it didn't work.